lib.rs

```diff
@@ -284,15 +338,17 @@ impl ThreadPool {
     /// ```
     /// use threadpool::ThreadPool;
     ///
-    /// let num_threads = 10;
-    /// let pool = ThreadPool::new(num_threads);
-    /// for _ in 0..num_threads {
+    /// let pool = ThreadPool::new(4);
+    /// for n in 0..10 {
     ///     pool.execute(move || {
-    ///         panic!()
+    ///         if n % 2 == 0 {
+    ///             panic!()
```

Comment:
Maybe add a comment like "simulating panic" so the test output does not look that grimm